test/index.js

```diff
@@ -110,6 +110,20 @@ describe('with Express', () => {
       .end(done);
   });
 
+  it('should parse accept-language header (setting priority using any case)', done => {
```

Comment:
Can you please move this test up? It doesn't need to be tested with Express.
[This test](https://github.com/smhg/express-locale/blob/713f02853b3d661c824475cd602da89a44e22b23/test/index.js#L40) seems to be a good example.